examples/va/nfv/hci-ovs-dpdk-sriov/control-plane/nncp/values.yaml

```diff
@@ -0,0 +1,209 @@
+---
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: network-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  # nodes
+  node_0:
+    name: ostest-master-0
+    internalapi_ip: 172.17.0.5
+    tenant_ip: 172.19.0.5
+    ctlplane_ip: 192.168.122.10
+    storage_ip: 172.18.0.5
+  node_1:
+    name: ostest-master-1
+    internalapi_ip: 172.17.0.6
+    tenant_ip: 172.19.0.6
+    ctlplane_ip: 192.168.122.11
+    storage_ip: 172.18.0.6
+  node_2:
+    name: ostest-master-2
+    internalapi_ip: 172.17.0.7
+    tenant_ip: 172.19.0.7
+    ctlplane_ip: 192.168.122.12
+    storage_ip: 172.18.0.7
+
+  # networks
+  ctlplane:
+    dnsDomain: ctlplane.example.com
+    subnets:
+      - allocationRanges:
+          - end: 192.168.122.120
+            start: 192.168.122.100
+          - end: 192.168.122.200
+            start: 192.168.122.150
+        cidr: 192.168.122.0/24
+        gateway: 192.168.122.1
+        name: subnet1
+    prefix-length: 24
+    iface: enp7s0
+    mtu: 1500
```

Comment:
Should we use `mtu: 9000` here like we do in HCI VA [1]?

[1] https://github.com/openstack-k8s-operators/architecture/blob/cfbcd92551fa8b942c3ac05043f774a9f6d4aab4/examples/va/hci/control-plane/nncp/values.yaml#L44

Reply:
This was a copy/paste from nfv templates, I have updated to 9000. I think we need to update in the other nfv templates too 
https://github.com/openstack-k8s-operators/architecture/blob/main/examples/va/nfv/ovs-dpdk-sriov/nncp/values.yaml#L44